SU2_CFD/src/solvers/CFEASolver.cpp

```diff
@@ -829,7 +862,7 @@ void CFEASolver::Compute_StiffMatrix(CGeometry *geometry, CNumerics **numerics,
     for(auto color : ElemColoring) {
 
       /*--- Chunk size is at least OMP_MIN_SIZE and a multiple of the color group size. ---*/
-      SU2_OMP_FOR_DYN(roundUpDiv(OMP_MIN_SIZE, ColorGroupSize)*ColorGroupSize)
+      SU2_OMP_FOR_DYN(nextMultiple(OMP_MIN_SIZE, color.groupSize))
```

Comment:
In a number of places I did some replacing to make this operation more expressive.